app/assets/javascripts/application.js

```diff
@@ -11,6 +11,30 @@
 // about supported directives.
 //
 //= require jquery
+//= require jquery.turbolinks
 //= require jquery_ujs
 //= require turbolinks
+//= require bootstrap-sprockets
 //= require_tree .
+
+$(document).ready(function() {
+
+  $('.dropdown-toggle').on('mouseenter', function() {
+    $('ul .dropdown-menu').slideDown('fast');
+  });
+
+  $('.dropdown-toggle').on('mouseleave', function() {
```

Comment:
Here we're doing a look up of `.dropdown-toggle` twice by calling `$('.dropdown-toggle')`. We could speed this up if we say this following

```js
var $dropdownToggle = $('.dropdown-toggle');

$dropdownToggle.on('mouseenter' ...
$dropdownToggle.on('mouseleave' ...
```

👪 👪 👪 👪 👪 👪 👪 👪 👪 👪 👪 👪 👪 👪 👪 👪 👪 👪 👪 👪 👪 👪 👪 👪 👪 👪 👪 👪